test/index.test.js

```diff
@@ -22,6 +28,31 @@ t.test('handler', async t => {
     t.matchSnapshot({ dynamo: t.context.dynamo, sqs: t.context.sqs })
   })
 
+  t.test('indexes a new car file but not invalid blocks', async t => {
+    const bytes = new Uint8Array([1, 2, 3])
+    const hash = await sha256.digest(bytes)
+    const cid = CID.create(1, raw.code, hash)
+    bytes[bytes.length - 1] = bytes[bytes.length - 1] + 1 // mangle a byte
+    const { writer, out } = CarWriter.create([cid])
+    writer.put({ cid, bytes })
+    writer.close()
+
+    const carParts = []
+    for await (const part of out) {
+      carParts.push(part)
+    }
+    const carBytes = concat(carParts)
```

Comment:
Not blocking...you could probably save adding that dependency just for this...

```suggestion
    const carBytes = new Uint8Array(await new Blob(carParts).arrayBuffer())
```

Reply:
we run in old Node, so we would also need to install Blob.